core/bin/dust_api.rs

```diff
@@ -2211,7 +2254,7 @@ fn main() {
             get(databases_tables_list),
         )
         .route(
-            "/projects/:project_id/data_sources/:data_source_id/databases/:database_id/tables/:table_id",
+            "/projects/:project_id/data_sources/:data_source_id/databases/:database_id/tables/:table_id/rows",
```

Comment:
I may have missed it but should this be /rows/:row_id?

Reply:
No because you can batch upsert the rows (so you actually pass a map of {row_id -> content}. I could have done 2 endpoints, but not sure if really useful.

Reply:
Yep understood from the other PR :+1: